cg/meta/workflow/balsamic.py

```diff
@@ -1,181 +1,25 @@
-        """
-        # Fetch data for creating a MIP config file
-        data = self.build_config(family_obj)
-
-        # Validate and reformat to MIP config format
-        config_data = self.tb.make_config(data, pipeline)
-
-        return config_data
+    @staticmethod
+    def get_deliverables_file_path(case_id, root_dir):
+        """Generates a path where the Balsamic deliverables file for the case_id should be
+        located"""
+        return Path.joinpath(root_dir, case_id, "delivery_report", case_id + ".hk")
 
-    def build_config(self, family_obj: models.Family) -> dict:
-        """Fetch data for creating a MIP config file."""
-        data = {
-            "case": family_obj.internal_id,
-            "default_gene_panels": family_obj.panels,
-            "samples": [],
-        }
-        for link in family_obj.links:
-            sample_data = {
-                "sample_id": link.sample.internal_id,
-                "analysis_type": link.sample.application_version.application.analysis_type,
-                "sex": link.sample.sex,
-                "phenotype": link.status,
-                "expected_coverage": link.sample.application_version.application.sequencing_depth,
-            }
-            if sample_data["analysis_type"] in ("tgs", "wes"):
-                if link.sample.capture_kit:
-                    # set the capture kit from status: key or custom file name
-                    mip_capturekit = CAPTUREKIT_MAP.get(link.sample.capture_kit)
-                    sample_data["capture_kit"] = mip_capturekit or link.sample.capture_kit
-                else:
-                    if link.sample.downsampled_to:
-                        self.LOG.debug(f"{link.sample.name}: downsampled sample, skipping")
-                    else:
-                        try:
-                            capture_kit = self.lims.capture_kit(link.sample.internal_id)
-                            if capture_kit is None or capture_kit == "NA":
-                                self.LOG.warning(
-                                    f"%s: capture kit not found", link.sample.internal_id
-                                )
-                            else:
-                                sample_data["capture_kit"] = CAPTUREKIT_MAP[capture_kit]
-                        except HTTPError:
-                            self.LOG.warning(f"{link.sample.internal_id}: not found (LIMS)")
-            if link.mother:
-                sample_data["mother"] = link.mother.internal_id
-            if link.father:
-                sample_data["father"] = link.father.internal_id
-            data["samples"].append(sample_data)
-        return data
+    @staticmethod
+    def get_config_path(root_dir, case_id: str) -> Path:
+        """Generates a path where the Balsamic config for the case_id should be located"""
+        return Path.joinpath(root_dir, case_id, case_id + ".json")
 
     @staticmethod
     def fastq_header(line):
```

Comment:
Shoudl this not be in a more general fastq.py module? This is not really related to Balsamic but to fastq file format